src/main/java/io/connect/scylladb/ScyllaDbSinkConnectorConfig.java

```diff
@@ -358,7 +358,7 @@ public static ConfigDef config() {
             .define(
                     SECURITY_ENABLE_CONFIG,
                     ConfigDef.Type.BOOLEAN,
-                    true,
+                    false,
```

Comment:
Why is this being modified to non-default? Does that have to do with address translation?

Reply:
The default value for this config is false which was mistakenly set to true.